package.json

```diff
@@ -26,7 +26,7 @@
     "compile:sdk": "tsc --outDir build",
     "compile:cypress": "tsc --noEmit --types cypress,node $(find cypress -not -path 'cypress/application/*' -type f -name *.ts)",
     "compile:jest": "tsc --noEmit --types jest,node $(find jest -type f -name *.ts)",
-    "documentation:copy": "cp node_modules/@mxenabled/widget-post-message-definitions/docs/web-sdk-generated.md docs/widget_callback_props.md",
+    "documentation": "cp node_modules/@mxenabled/widget-post-message-definitions/docs/web-sdk-generated.md docs/widget_callback_props.md",
```

Comment:
`npm run documentation` used to be `npm run documentation:copy` -- we can reintroduce the namespacing if we have more "documentation" commands in the future.